app/controllers/help_controller.rb

```diff
@@ -0,0 +1,15 @@
+class HelpController < ApplicationController
+  layout 'help'
+
+  def index
+    dir = Rails.root.join('app', 'views', 'help', '*.md')
+    files = Dir.glob(dir)
+    @pages = files.map{|file| File.basename(file, '.md') }.sort
+  end
+
+  def show
+    page = params[:id]
+    # TODO make sure this isn't a security hole
```

Comment:
Probably best to use a whitelist. Maybe pull out the listing logic in `index` into a shared function (or class var) to get such a list. Only render if `page` is a member of that list (404 otherwise)?